bin/plugin/commands/changelog.js

```diff
@@ -513,7 +513,7 @@ function removeFeaturePrefix( text ) {
  * @type {Array<WPChangelogNormalization>}
  */
 const TITLE_NORMALIZATIONS = [
-	createOmitByLabel( [ 'Mobile App Android/iOS' ] ),
+	createOmitByLabel( [ 'Mobile App - i.e. Android or iOS' ] ),
```

Comment:
Could we make a fuzzy matcher here so that anything with "Mobile App" get's removed. Not a blocker though.

Reply:
Yeah, I thought of that, but only after opening the PR 😅 .

 I've added a label prefix checker instead, see my updated PR description 🙂